dist/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<!-- fontawesome -->
+		<script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+		<title>Movie App</title>
+	<script defer src="main.js"></script></head>
+	<body>
+		<header id="header"></header>
+		<main>
+			<h1>Trending Movies</h1>
+			<div class="movies"></div>
+			<!-- <section>
+				<div class="comment-section">
+					
+				</div>
+			</section> -->
```

Comment:
I  strongly recommend you remove all comments in the code. Using comments does not make the code clean

Reply:
Thanks for the review 😊